packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -120,10 +120,12 @@ export interface IPersistedCache {
     updateUsage(entry: ICacheEntry, seqNumber: number): void;
 
     /**
-     * Removes all the entries from the cache for given docId.
-     * @param docId - DocId for which all entries needs to be deleted from cache.
+     * Removes the entries from the cache for given parametres.
+     * @param file - file entry to be deleted.
+     * @param type - Type of entry to be deleted for a given file.
+     * @param key - Specific entry containing this key to be deleted.
      */
-    removeAllEntriesForDocId(docId: string): Promise<void>;
+    removeEntries(file?: IFileEntry, type?: CacheContentType, key?: string): Promise<void>;
```

Comment:
Based on latest discussion with SPO, I think we can safely remove type & key.
Though type might be still useful for R11S in the future if they use same workflow, as in their world blobs are unique.
Key - not sure, do we have any (future) scenarios where it might be needed? If not, might be worth removing from API

Reply:
Yeah based on the dicussions we can just keep the file entry.